### ISSUE TITLE
MP4 not supported

### ISSUE THREAD

**Issue by zc99** [joheras/CLoDSA#15]
![image](https://user-images.githubusercontent.com/69033926/89017302-a9ebce80-d34c-11ea-9b2b-5acac9438360.png)

Which video format that is supported for video augmentation?

**Comment by joheras** [comment]
Currently, it only supports the AVI format.

**Comment by zc99** [comment]
Any other way to do augmentation on videos?

**Comment by joheras** [comment]
What do you mean? Do you want to have other formats?

**Comment by zc99** [comment]
Ya, means is there a way to do augmentation on video of other format. 
If I manually convert MP4 to avi format, does it work?

**Comment by joheras** [comment]
Yes, converting from mp4 to avi, and then applying clodsa will work. I will
include in the future other formats like mp4.

**Comment by zc99** [comment]
Thank you. I will try on it.

**Comment by joheras** [comment]
Let me know if you need further help
